uploadfs.js

```diff
@@ -65,16 +64,45 @@ function Uploadfs() {
       self.cdn = options.cdn;
     }
 
+
     // Load image backend
-    self._image = options.image;
+     self._image = options.image;
+    const processors = [ 'imagemagick', 'jimp' ];
     if (typeof (self._image) === 'string') {
-      self._image = require('./lib/image/' + self._image + '.js')();
+      if (processors.includes(self._image)) {
+        console.error('We now recommend the use of the sharp.js library for image processing. As fallback, if sharp is not installed we will attempt to use imagemagick instead.');
```

Comment:
"sharp.js library (included)"

"If sharp installation fails..."

Don't want them thinking they have to install things.

Reply:
Changed.